DMTN-272.tex

```diff
@@ -49,7 +50,184 @@
+
+\begin{figure}[h]
+  \centering
+  \includegraphics[width=0.7\textwidth]{tract_templates.png}
+  \caption{Templates generated based on raw images from a single tract. The white chessboard in the background is the spatial spread of the detectors (a single visit/exposure), whereas the area ``full of stars'' represents a single tract in the sky.}
+  \label{fig:tract_templates}
+\end{figure}
+
+%----------------------------------------------------
+\clearpage
+\section{Evaluation Results}
+
+\subsection{The ``PCW 2022'' network and data}
+The data-set used for training this model, DL Dataset v0.1.1, consists of \texttt{78222} triplets, out of which \texttt{2591} triplets ($\sim 3\%$) are \texttt{positive}s.
+
+\begin{figure}[h]
+  \centering
+  \includegraphics[width=0.5\textwidth]{precrec___checkpoint_epoch0000087_0200000____rbdata_data_npy_data_gt1M___posw_30.png}
+  \caption{No sigma-thresholding applied on the input sources (diffim output)}
+  \label{fig:tract_templates}
+\end{figure}
+
+\begin{figure}[h]
+  \centering
+  \includegraphics[width=0.5\textwidth]{precrec___checkpoint_epoch0000087_0200000____rbdata_data_npy_data_gt1M-5sigma___posw_30.png}
+  \caption{Only sources with $snr > 5\sigma$}
+  \label{fig:tract_templates}
+\end{figure}
+
+\begin{figure}[h]
+  \centering
+  \includegraphics[width=0.5\textwidth]{precrec___checkpoint_epoch0000087_0200000____rbdata_data_npy_data_gt1M-6sigma___posw_30.png}
+  \caption{Only sources with $snr > 6\sigma$}
+  \label{fig:tract_templates}
+\end{figure}
+
+% ----------------------------------------------------
+\clearpage
+\subsection{New Dataset}
+\label{sec:new_dataset}
+The new dataset (DL v0.1.2) is essentially the same as the previous version (v0.1.1), only regenerated with a more recent version of the stack (\texttt{w\_2023\_38}). The number of images is relatively higher though, in the new version: there are \texttt{183750} triplets, out of which \texttt{10147} ($\sim 5\%$) are positives.
+
+We used a $95\%-5\%$ training-validation split, which leaves us with \texttt{9198} validation triplets (with \texttt{533} $\simeq 5\%$ positives). Below evaluation results are based on a snapshot of the network after 255K iterations (937 epochs).
+
+
+\begin{figure}[h]
+  \centering
+  \includegraphics[width=0.4\textwidth]{precrec_13-resnet50-FullAugmentation-scratch-B64__0255000__npy_data_0.1.2-0sigma_256by256__posw_1.png}
+  \includegraphics[width=0.4\textwidth]{confmat_13-resnet50-FullAugmentation-scratch-B64__0255000__npy_data_0.1.2-0sigma_256by256__posw_1.png}
+  \includegraphics[width=0.4\textwidth]{roc_13-resnet50-FullAugmentation-scratch-B64__0255000__npy_data_0.1.2-0sigma_256by256__posw_1.png}
+  \includegraphics[width=0.4\textwidth]{det_13-resnet50-FullAugmentation-scratch-B64__0255000__npy_data_0.1.2-0sigma_256by256__posw_1.png}
+  \caption{Evaluation results based on the whole dataset.}
```

Comment:
"Whole dataset" needs some explanation here: what specifically differs between this and the SNR > 5 selection?

Reply:
"The dataset" in this section refers to the "DL Dataset v0.1.1", as pointed out in the text. It is based on all the sources reported by diffim. So no SNR thresholding has been applied.